### ISSUE TITLE
high efficiency recording extractor

### ISSUE THREAD

**Issue by lfrank** [magland/sortingview#98]
Currently sortingview doesn't include the labbox-ephys high efficiency recording extractor.  It would be helpful if this was added so we can try to speed up the process of writing the recording extractor to a file.

**Comment by lfrank** [comment]
I found the extractor, but it's not included in the extractor __init__.py, so I can't import it. Could you fix that?

**Comment by magland** [comment]
@lfrank This is now fixed in version 0.2.38 (on pypi)

So you can do:

```python
import sortingview as sv
import kachery_client as kc

recording = ...

X = sv.H5RecordingExtractorV1.write_recording(recording=recording, h5_path='/some/path.h5')

# Load it at a later time
Y = sv.H5RecordingExtractorV1(h5_path='/some/path.h5')

# The recommended/preferred method is to use LabboxEphysRecordingExtractor
recording_uri = kc.link_file('/some/path.h5')
Z = sv.LabboxEphysRecordingExtractor({
    'recording_format': 'h5_v1',
    'data': {'h5_uri': recording_uri}
}
```

You can examine the source code here:
https://github.com/magland/sortingview/blob/09124de0a99a44fad0ff8580cdc0dde379a8be56/src/python/sortingview/extractors/labboxephysrecordingextractor.py#L212-L387

and here:
https://github.com/magland/sortingview/blob/09124de0a99a44fad0ff8580cdc0dde379a8be56/src/python/sortingview/extractors/h5extractors/h5recordingextractorv1.py#L12-L48

**Comment by lfrank** [comment]
@magland: I’ve tried 0.2.38 and 0.3.0 and neither seems to have sv. H5RecordingExtractorV1. Can you check this on your end?

**Comment by magland** [comment]
oops, i exported the H5SortingExtractorV1, but not H5RecordingExtractorV1... this will be fixed on the next release.

**Comment by magland** [comment]
@lfrank This should be fixed in 0.3.1 on pypi. Once you confirm you can close this issue.
